libs/deal-ticket/src/components/info-market.tsx

```diff
@@ -371,7 +474,9 @@ const Row = ({
         dtClassName={className}
         ddClassName={className}
       >
-        {startCase(t(field))}
+        <Tooltip description={tooltipMapping[field]} align="start">
+          <div>{startCase(t(field))}</div>
```

Comment:
nitpick (non-blocking): you can make this div focusable by adding `tabIndex={-1}`. This will allow to be seen a tooltip content on mobile (when no mouse events) after tap on it.

Reply:
nice, added